data_override/README.MD

```diff
@@ -24,6 +24,13 @@ If it is desired to interpolate the data to a region of the model grid. The foll
 - **lat_start:** The starting longitude in the same units as the grid data in the file
 - **lon_end:** The ending longitude in the same units as the grid data in the file
 
+If it is desired to use multiple(3) input netcf files instead of 1. The following **optional** keys are available.
+- **is_multi_file:** Set to `True` is using the multi-file feature
+- **prev_file_name:** The name of the first file in the set
+- **next_file_name:** The name of the third file in the set
+
+Note that **file_name** must be the second file in the set. **prev_file_name** and **next_file_name** are required if **is_multi_file** is set to `True`
```

Comment:
and/or

I have updated the code so that we can use only prev or next, still needs 2 separators in the old diag_table though